
bool EMERGENCY_EMC
bool EMERGENCY_BUMPER
bool EMERGENCY_PROXIMITY
bool EMERGENCY_SHARED_EMC
